### ISSUE TITLE
bug: the signature set in gmail doesn't appear when composing and sending mail with wmail

### ISSUE THREAD

**Issue by bertrandmartin01** [Thomas101/wmail#52]

**Comment by bertrandmartin01** [comment]
@Thomas101 any chance this can be looked into? it kind of prevents using professional accounts

thanks a lot and once again amazing work!

**Comment by Thomas101** [comment]
Hi, I'll take a look and see what I can find :). I'll add this to 1.1.6

**Comment by bertrandmartin01** [comment]
thanks a lot thomas

**Comment by Thomas101** [comment]
Hi, I've taken a look at this and can't seem to reproduce it. I set my signature in gmail, then added the account in wmail and it worked just the same. Tried the same with inbox and that was the same. The only thing I did notice is that inbox doesn't show the signature at the bottom of the email when you're composing, but does add it once you've sent it.

If you open up your mailbox in wmail and check the signature settings are they enabled?

**Comment by mykecheckonetwo** [comment]
If this is what I’m thinking of, Inbox doesn’t actually “show” the signature. Its part of it’s build. It does send it though. 

On Wed, Jan 27, 2016 at 1:19 PM Thomas Beverley

<
mailto:Thomas Beverley <notifications@github.com>
> wrote:

Hi, I've taken a look at this and can't seem to reproduce it. I set my signature in gmail, then added the account in wmail and it worked just the same. Tried the same with inbox and that was the same. The only thing I did notice is that inbox doesn't show the signature at the bottom of the email when you're composing, but does add it once you've sent it.

If you open up your mailbox in wmail and check the signature settings are they enabled?
